php-checks/src/main/java/org/sonar/php/checks/HardCodedCredentialsCheck.java

```diff
@@ -40,9 +44,11 @@
 
   public static final String KEY = "S2068";
   private static final String MESSAGE = "'%s' detected in this variable name, review this potentially hardcoded credential.";
+  private static final String MESSAGE_ARGUMENTS = "detected string in password argument, review this potentially hardcoded credential.";
```

Comment:
I would change the message to "Review this hardcoded credential."
Sorry, I didn't see that before.

Reply:
Changed